certs/service.go

```diff
@@ -171,17 +150,17 @@ func (cs *certsService) RevokeCert(ctx context.Context, token, thingID string) (
 	}
 	thing, err := cs.sdk.Thing(thingID, token)
 	if err != nil {
-		return revoke, errors.Wrap(errFailedCertRevocation, err)
+		return revoke, errors.Wrap(ErrFailedCertRevocation, err)
```

Comment:
Revoke seems to be a simple wrapper around `time.Time`. Since it's unlikely its content will ever change, this wrapper seems to be unnecessary; you can return time directly.

Reply:
more or less I copied response from `vault` https://www.vaultproject.io/api/secret/pki#sample-request-14

Reply:
You can (and need) to wrap time in API layer, but there is no need to wrap it in the service method.